src/main/java/frc/robot/subsystems/OuttakeSubsystem.java

```diff
@@ -41,10 +42,10 @@ public OuttakeSubsystem() {
     m_shooterMotorI.follow(m_shooterMotorII, true);
     m_shooterMotorI.setIdleMode(IdleMode.kBrake);
     m_shooterMotorII.setIdleMode(IdleMode.kBrake);
-
-    m_shooterMotorI.setSmartCurrentLimit(90);
-    m_shooterMotorII.setSmartCurrentLimit(90);
-    m_takeNoteMotor.setSmartCurrentLimit(90);
+    m_shooterMotorI.setSmartCurrentLimit(OUTTAKE_CURRENT_LIMITS);
+    m_shooterMotorI.setSmartCurrentLimit(OUTTAKE_CURRENT_LIMITS);
+    m_shooterMotorII.setSmartCurrentLimit(OUTTAKE_CURRENT_LIMITS);
+    m_takeNoteMotor.setSmartCurrentLimit(OUTTAKE_CURRENT_LIMITS);
```

Comment:
You never set the pivot motor's current limits. Also, there should be a different constant for the pivot motor's current limit.